stock_orderpoint_manual_procurement/wizards/make_procurement_orderpoint_view.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <!--  Make Procurement -->
+
+    <record id="view_make_procurment_buffer_wizard" model="ir.ui.view">
+        <field name="name">Request Procurement</field>
+        <field name="model">make.procurement.orderpoint</field>
+        <field name="arch" type="xml">
+            <form string="Procurement Request">
+                <p class="oe_gray">
+                    Use this assistant to generate a procurement request for this
+                    stock buffer. According to the product configuration,
+                    this may trigger a draft purchase order, a manufacturing
+                    order or a transfer picking.
+                </p>
+                <group name="items" string="Items">
+                    <field name="item_ids" nolabel="1">
+                        <tree string="Items" nocreate="1" editable="top">
+                            <field name="orderpoint_id" invisible="True"/>
+                            <field name="warehouse_id" groups="stock.group_stock_multi_locations"/>
+                            <field name="location_id" groups="stock.group_stock_multi_locations"/>
+                            <field name="product_id"/>
+                             <field name="qty" groups="stock_orderpoint_manual_procurement.group_change_orderpoint_procure_qty"/>
+                            <field name="uom_id" groups="product.group_uom"/>
+                        </tree>
+                    </field>
+                </group>
+                <footer>
+                    <button name="make_procurement" string="Execute"
+                            type="object"  class="oe_highlight"  />
+                    or
+                    <button string="Cancel" class="oe_link" special="cancel" />
```

Comment:
Should be updated to the latest changes, example from the account module:

```xml
                    <footer>
                        <button string="Reconcile" name="trans_rec_reconcile" type="object" default_focus="1" class="btn-primary"/>
                        <button string="Cancel" class="btn-default" special="cancel"/>
                    </footer>
```

The `or` must be moved, the class attribute change on both buttons and there is a default_focus on the primary button.

Reply:
Corresponded. Thanks.